administrator-manual/en/fail2ban.rst

```diff
@@ -0,0 +1,160 @@
+========
+Fail2ban
+========
+
+Fail2ban scans log files (e.g. /var/log/apache/error_log) and bans IPs that show the malicious signs – too many password failures, seeking for exploits, etc. Generally Fail2Ban is then used to update firewall rules to reject the IP addresses for a specified amount of time, although any arbitrary other action (e.g. sending an email) could also be configured. Out of the box Fail2Ban comes with filters for various services (apache, courier, ssh, etc).
+
+Fail2Ban is able to reduce the rate of incorrect authentications attempts however it cannot eliminate the risk that weak authentication presents. Configure services to use only two factors or public/private authentication mechanisms if you really want to protect services.
+
+Installation
+============
+
+Install from the Software Center or use the command line: ::
+
+  yum install nethserver-fail2ban
+
+
+Settings
+========
+
+Fail2ban is configurable in the security category of the server-manager. Most of settings can be changed in the ``Configuration`` tab, only really advanced settings must be configured by the terminal. The ``Ban status`` tab displays the statistic and the banned IPs.
+
+You can also activate changes and restart the fail2ban service by: ::
+
+  signal-event nethserver-fail2ban-save
+
+Jails
+-----
+
+A jail is enabled and start to protect a service when you install a new module, the relevant jail (if existing) is automatically activated by the event 'runlevel-adjust' which is launched at the end of the yum process.
```

Comment:
The user doesn't know what is the runlevel-adjust.
I'd change it  to something like:
"the relevant jail is automatically activated then new packages are installed"